guides/README.md

```diff
@@ -0,0 +1,27 @@
+# Rails Guides Redesign 2024
+
+## About the Project
+
+The Rails Guides Visual Refresh occured in Q1 2024, and was intended to bring the visual style of the guides inline with the rubyonrails.org site. 
+
+## Editing Depedencies
+
+The editing files for the Guides rebuild reside in `stylesrc` and use SCSS to improve developer experience. The code base relies on `include_media` (https://eduardoboucas.github.io/include-media/) to enable inline media-queries adjustments. We've also relied on the standard `normalize.css` (https://necolas.github.io/normalize.css/) to help bring all browsers together.
+
+## Building the Guides in Development
+
+To generate new guides into static files, type `rake guides:generate` from inside the `guides` folder. If you make changes to the HTML or ERB, you'll need to remove the "output" directory before running this command. The master SCSS files (style.scss, highlight.scss) will compile as part of this process. 
+
+## FAQ
```

Comment:
I think there's some useful context in here, but this isn't the right place for it to live: future visitors would expect `guides/README` to be a readme about the content of the guides, not the more rarely edited styles.

Reply:
Okay. There was no guides README before (it had been left behind with a note that it had been removed). I put this in as it took me a bit of digging to figure out how to build the guides the first time. I also wanted to explain some things to any who might want to contribute to the guides in the future. I expect that the content team would add additional detail here (w/r/t content of the guides) in their PR, but open to changes here.

Reply:
I figured we'd add more to the readme about the guides themselves later on as well.